src/test/java/task/TestDeadline.java

```diff
@@ -0,0 +1,29 @@
+package task;
+
+import duke.exception.DukeException;
+import duke.task.Deadline;
+import duke.util.DateUtil;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TestDeadline {
+
+    private Deadline deadline;
+
+    @BeforeEach
+    public void setUp() throws DukeException {
+        deadline = new Deadline("return a book", DateUtil.parse("1/12/2019 1800"));
+    }
+
+    @Test
+    public void toString_normal_noException() {
+        assertEquals(deadline.toString(), "[D][✘] return a book (by: 01/12/2019 1800)");
```

Comment:
This is not a major problem. However, according to the syntax for `assertEquals`, the expected output should be the first argument while the actual output should be the second argument. 😅